From 9f93d0801470c58e98a2b554efabeb3042bca215 Mon Sep 17 00:00:00 2001
From: Melker Narikka <meklu@meklu.org>
Date: Tue, 23 Oct 2012 23:03:37 +0300
Subject: [PATCH] Disabled debug mode.

---
 init.lua | 2 ++--
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.lua b/init.lua
index f0d1ff6..c393cdd 100644
--- a/init.lua
+++ b/init.lua
@@ -18,7 +18,7 @@ loveframes.config["DIRECTORY"]       = "libraries/loveframes"
 loveframes.config["DEFAULTSKIN"]     = "Blue"
 loveframes.config["ACTIVESKIN"]      = "Blue"
 loveframes.config["INDEXSKINIMAGES"] = true
-loveframes.config["DEBUG"]           = true
+loveframes.config["DEBUG"]           = false
 
 -- misc library vars
 loveframes.drawcount                 = 0
-- 
1.7.12

