From 5a59e529d58ed3750d73c9378b191ebe4ac9acaf Mon Sep 17 00:00:00 2001
From: Melker Narikka <meklu@meklu.org>
Date: Sun, 9 Mar 2014 16:59:48 +0200
Subject: [PATCH] Disabled debug mode.

---
 init.lua | 2 ++--
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.lua b/init.lua
index d2ff55e..ef987cd 100644
--- a/init.lua
+++ b/init.lua
@@ -15,7 +15,7 @@ loveframes.stage = "Alpha"
 
 -- library configurations
 loveframes.config = {}
-loveframes.config["DIRECTORY"] = nil
+loveframes.config["DIRECTORY"] = "libraries/loveframes"
 loveframes.config["DEFAULTSKIN"] = "Blue"
 loveframes.config["ACTIVESKIN"] = "Blue"
 loveframes.config["INDEXSKINIMAGES"] = true
1.9.0
